### ISSUE TITLE
WCAG 4.1.2: Nested interactive controls are not announced by screen readers (.auth-user)

### ISSUE THREAD

**Issue by maxvisser** [teamforus/forus#2307]
#### Issue
`Nested interactive controls are not announced by screen readers` ([nested-interactive](https://dequeuniversity.com/rules/axe/4.2/nested-interactive?application=msftAI))
#### Target application
[Uw tegoed](https://berkelland.staging.forus.io/vouchers)
#### Element path
.auth-user
#### Snippet
`<div class="auth-user" ng-click="$ctrl.openUserMenu($event)" role="button" tabindex="0">`
#### How to fix
    Fix any of the following:
      Element has focusable descendants
#### Environment
Microsoft Edge version 90.0.818.66
This accessibility issue was found using Accessibility Insights for Web 2.26.1 (axe-core 4.2.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
